Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -338,15 +367,87 @@ public function renderThumbnailsCommand(int $limit = null, bool $quiet = false)
                 break;
             }
         }
-        !$quiet && $this->output->progressFinish();
+    }
+
+    /**
+     * Render asset variants
+     *
+     * Loops over missing configured asset variants and renders them. Optional ``limit`` parameter to
+     * limit the amount of variants to be rendered to avoid memory exhaustion.
+     *
+     * If the re-render parameter is given, any existing variants will be rendered again, too.
+     *
+     * @param integer $limit Limit the amount of variants to be rendered to avoid memory exhaustion
+     * @param bool $quiet If set, only errors will be displayed.
+     * @param bool $recreate
+     * @return void
+     * @throws AssetVariantGeneratorException
+     * @throws IllegalObjectTypeException
+     */
+    public function renderVariantsCommand($limit = null, bool $quiet = false, bool $recreate = false): void
+    {
+        $resultMessage = null;
+        $generatedVariants = 0;
+        $configuredVariantsCount = 0;
+        $configuredPresets = $this->assetVariantGenerator->getVariantPresets();
+        foreach ($configuredPresets as $configuredPreset) {
+            $configuredVariantsCount += count($configuredPreset->variants());
+        }
+
+        $classNames = $this->reflectionService->getAllImplementationClassNamesForInterface(VariantSupportInterface::class);
+        foreach ($classNames as $className) {
+            /** @var AssetRepository $repository */
+            $repositoryClassName = $this->reflectionService->getClassSchema($className)->getRepositoryClassName();
+            $repository = $this->objectManager->get($repositoryClassName);
+
+            if (!method_exists($repository, 'findAssetIdentifiersWithVariants')) {
```

Comment:
Uh, there's no interface for that?!

Reply:
No, because… 🤷‍♂️

But yes, would have been good. Also, `Image` could get (some additional) interface for `replaceVariant()`, I guess.

Reply:
Hah, that interface would be `VariantSupportInterface`, that should be amended. A topic for the next major…